src/txadmission.cpp

```diff
@@ -699,9 +699,10 @@ bool ParallelAcceptToMemoryPool(Snapshot &ss,
     }
 
     uint32_t featureFlags = 0;
-    if (IsNov2018Activated(chainparams.GetConsensus(), chainActive.Tip()))
+    if (IsMay2020Enabled(chainparams.GetConsensus(), chainActive.Tip()))
     {
```

Comment:
If you're going to leave a stub you should maybe comment it out entirely and then add a comment so the next dev knows doesn't forget that this isn't just useless code to be removed.